recipes/default.rb

```diff
@@ -1,4 +1,9 @@
 include_recipe 'java'
+
+package 'openssl-devel' do
```

Comment:
this can be left as:

```package 'openssl-devel'```

Because ```:install``` is the default action for the package resource in chef. Not a big deal though.